### ISSUE TITLE
python3.6/site-packages/torch/serialization.py:292: UserWarning: Couldn't retrieve source code for

### ISSUE THREAD

**Issue by ada-dl** [sgrvinod/a-PyTorch-Tutorial-to-Image-Captioning#96]
After the first training epoch completes, I get these warnings for a variety of things, examples below: 

`python3.6/site-packages/torch/serialization.py:292: UserWarning: Couldn't retrieve source code for container of type Encoder. It won't be checked for correctness upon loading.
  "type " + obj.__name__ + ". It won't be checked "`

`python3.6/site-packages/torch/serialization.py:292: UserWarning: Couldn't retrieve source code for container of type Sequential. It won't be checked for correctness upon loading.
  "type " + obj.__name__ + ". It won't be checked "`

and a bunch of others, such as BatchNorm2d., ReLU etc. Could you let me know what these mean? Is it because we are using the pretrained Resnet as our Encoder and we don't have the source code for it? I would really appreciate if someone could help me understand these warnings. Thank you.

**Comment by kmario23** [comment]
Long answer short: you can ignore these warnings. If you're looking for a discussion, here is it: [got-warning-couldnt-retrieve-source-code-for-container](https://discuss.pytorch.org/t/got-warning-couldnt-retrieve-source-code-for-container/7689)

**Comment by ada-dl** [comment]
thank you
